### DISCUSSION TITLE
Error while fetching app health: client_error: client error: 401

### DISCUSSION THREAD

**Discussion by 631068264** [kiali/kiali#6022] [Q&A]
I install this by rancher app store ,  use Istio app

<img width="1410" alt="image" src="https://user-images.githubusercontent.com/8144089/231367972-30612ad5-d6ce-4301-9e2f-b761d26b179c.png">

```
/k8s/clusters/xxxx/api/v1/namespaces/istio-system/services/http:kiali:20001/proxy/kiali/api/namespaces/default/health?type=app&rateInterval=60s

status code 503

{"error":"Error while fetching app health: client_error: client error: 401"}

```

Can't get some namespaces health by api， but other namespaces can

Kiali 1.44  istio 1.12.6

Also can't get graph
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/8144089/231369705-6858f426-d420-46ad-bb62-da08186c9706.png">

```
/k8s/clusters/xxx/api/v1/namespaces/istio-system/services/http:kiali:20001/proxy/kiali/api/namespaces/graph?duration=60s&graphType=versionedApp&includeIdleEdges=false&injectServiceNodes=true&boxBy=app&appenders=deadNode,sidecarsCheck,serviceEntry,workloadEntry,istio,healthConfig&rateGrpc=requests&rateHttp=requests&rateTcp=sent&namespaces=istio-example

status code 503



{"error":"client_error: client error: 401"}

```
But other  tap can use  such as Applications
<img width="1420" alt="image" src="https://user-images.githubusercontent.com/8144089/231370255-12362791-893f-455b-a42e-495de9873acb.png">

**Comment by jmazzitelli** [comment]
(_moved to Q&A discussion_)

First, this is a really old Kiali (1.44) so it is possible if this is a bug it likely was fixed in a newer version. FYI: Kiali 1.44, along with Istio 1.12 has been out of support for a while (https://kiali.io/docs/installation/installation-guide/prerequisites/#version-compatibility) so any issues in that old of a Kiali will not be patched.

You mention Rancher and that long URL (/k8s/clusters/xxx/api/v1/namespaces/istio-system/services/http:kiali:20001/proxy/kiali/api/namespaces/graph...) which looks familiar. I seem to recall a year or so ago that Rancher installs had an issue because of the proxy they were using. Do a search in the Kiali github and search for this - there may be a workaround (I can't remember if the problem was solved or not, but this does sound familiar to me).
